### PR TITLE
Hotfix/repeat concat

### PR DESCRIPTION
**Pull request by scottnath**
Fixes a bug where repeatables were concat-ing each other's data

---
Resolves #https://github.com/punchcard-cms/punchcard/issues/446

`DCO 1.1 Signed-off-by: Scott Nath <github@scottnath.com>`

### CODE REVIEW

File: lib/form/html.js [hunk 1]
```diff
@@ -189,7 +189,7 @@ const renderer = (type, errors) => {
 
           // Adds delete button if instances are greater than minimum
           if (input.inputs.length > input.repeatable.min) {
-            render += `<button type="button" class="delete--button" id="${input.id}--delete--${index}">Delete</button>`;
+            render += `<button type="button" class="form--delete" id="${input.id}--delete--${index}">Delete</button>`;
```
Comment (Snugug) [inline lib/form/html.js]: Is the JS looking for `form--delete` or `delete--button`? In the [HTML generation in Content Types](https://github.com/punchcard-cms/content-types/blob/master/lib/form/html.js#L192) it's adding `delete--button` but it _looks_ like the [JS is looking for `form--delete`](https://github.com/punchcard-cms/content-types/blob/master/lib/form/js/repeatable.js#L196). 

Can you confirm which is which and update whichever is wrong accordingly? It also looks like we may need to refactor that JS a bit.

Comment (scottnath) [inline lib/form/html.js]: Well, that's why this line is changed. It should be `form--delete`. The link you have to `HTML generation in Content Types` is the link to this line...so, that (this) line is, ya know, already updated.

No other references to `delete-button` exist that I know of.

**Comment by scottnath** [comment]
@Snugug answered. Confused if I'm missing something, but I think this PR is already GTG.

File: lib/form/html.js (hunk 1 above)
Comment (Snugug) [inline lib/form/html.js]: HAHA, oh wow, ok. Go me.

**Pull request merged**